esp/src/src-react/components/GetDFSCSV.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import { DaliService } from "@hpcc-js/comms";
+import { scopedLogger } from "@hpcc-js/util";
+
+const logger = scopedLogger("src-react/components/GetDFSCSV.tsx");
+
+const daliService = new DaliService({ baseUrl: "" });
+
+
+interface GetDFSCSVProps {
+
+}
+
+export const GetDFSCSV: React.FunctionComponent<GetDFSCSVProps> = ({
+
+}) => {
+
+    const [dfsCsv, setDfsCsv] = React.useState<string>("");
+
+    React.useEffect(() => {
+        daliService.GetDFSCSV({LogicalNameMask:""}).then(( response ) => {
+            setDfsCsv(response.Result);
+        }).catch(err => logger.error(err));
+    }, []);
+
+    return <div>
+        {dfsCsv}
```

Comment:
I'm not sure what the data looks like coming back from these various requests, but you might use one of the Editor components in .../components/SourceEditor.tsx to display the contents. You can look at .../components/Configuration.tsx for an example of that for XML.